### PR TITLE
[EOC-608] User friendly names in demo mode (icons)

### PR DESCRIPTION
**Pull request by alemikolo**

### CODE REVIEW

File: server/seed/demoSeed/generateLists.js [hunk 1]
```diff
@@ -16,13 +16,18 @@ const generateLists = (demoUserId, userIds, cohortIds, items) => [
     favIds: [demoUserId],
     isArchived: false,
     isDeleted: false,
-    items: generateItems(demoUserId, userIds),
+    items: generateItems(demoUserId, userIds, [
+      '🍅 Tomato',
+      '🥓 Bacon',
+      '🧅 Onion',
```
Comment (adamklepacz) [inline server/seed/demoSeed/generateLists.js]: Onion emoji missing

Comment (alemikolo) [reply server/seed/demoSeed/generateLists.js]: Isn't displayed? I  see Onion even on safari

**Comment by alemikolo** [comment]
I have no errors and clean console... Did you update dependencies? Just in case I merge master to this branch reinstall dependencies and there are no errors...

File: server/seed/demoSeed/generateLists.js (hunk 1 above)
Comment (adamklepacz) [reply server/seed/demoSeed/generateLists.js]: nope, No onion in the browser. 
Maybe replace with something diffrent

**Pull request merged**